hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -56,6 +56,29 @@ export const getTeamOrders = createAsyncThunk<
     }
 });
 
+export const updateOrder = createAsyncThunk<
+    OrderInTable,
+    number,
+    { state: RootState; rejectValue: RejectValue; dispatch: AppDispatch }
+>(`${orderReducerName}/updateOrder`, async (order_id, { rejectWithValue }) => {
+    try {
+        const data = {
```

Comment:
you don't need the request property here, PATCH requests are partial updates so status in the request body is enough.

Also setting request to "string" would not be a good idea lol, check out the order model in hardware/models.py to learn more about what the request property does.